ckan/config/middleware/flask_app.py

```diff
@@ -14,7 +37,18 @@ def make_flask_stack(conf, **app_conf):
     """ This has to pass the flask app through all the same middleware that
     Pylons used """
 
+    root = os.path.dirname(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Can you point me to the method on `pkg_resources` to use or an example? All my googling returned approaches like the one above.